draft-ietf-masque-h3-datagram.md

```diff
@@ -773,11 +458,8 @@ This registry initially contains the following entries:
 
 | Capsule Type                 |   Value   | Specification |
 |:-----------------------------|:----------|:--------------|
-| REGISTER_DATAGRAM_CONTEXT    | 0xff37a1  | This Document |
-| REGISTER_DATAGRAM            | 0xff37a2  | This Document |
-| CLOSE_DATAGRAM_CONTEXT       | 0xff37a3  | This Document |
-| DATAGRAM_WITH_CONTEXT        | 0xff37a4  | This Document |
-| DATAGRAM                     | 0xff37a5  | This Document |
+| DATAGRAM                     | 0xff37a0  | This Document |
+| RELIABLE_DATAGRAM            | 0xff37a5  | This Document |
```

Comment:
appreciate this is just the first proposal but the format changes and codepoint usage here seems breaking, sp probably requires a bump to the H3_DATAGRAM setting codepoint.

Reply:
Interestingly, these changes don't break backwards compatibility because the format is the same in the absence of the Sec-Use-Datagram-Contexts header. So we don't need to change the setting or codepoints - which is nice because it doesn't break existing WebTransport implementations.

Reply:
Right but previously the intermediary could manipulate DATAGRAM capsules with type 0xff37a5, and now the text says they can't do that because the semantics of the RELIABLE_DATAGRAM are different. Is the assumption that no one implemented an intermediary that procesed DATAGRAM before this change?

Reply:
actually, that's a problem with capsule protocol versioning not the H3 SETTING but I think my point stands

Reply:
Ah right, yes the (unstated) assumption was that no one has yet implemented an intermediary yet so we don't need to maintain backwards compatibility yet.